fetch.bs

```diff
@@ -760,7 +764,12 @@ fetch("https://victim.example/naïve-endpoint", {
     </div>
 
    <dt>Otherwise
-   <dd><p>Return false.
+   <dd>
+    <p>If <var>name</var> does not begin with the string "<code>sec-</code>", return false.
+
+    <p class=note>As all headers beginning with "<code>Sec-</code>" are <a>forbidden header
+    names</a>, we have some confidence that they're generated by the user agent, and not via APIs
+    that developers directly control.
```

Comment:
The other thing we should point out here is https://github.com/whatwg/fetch/pull/880#issuecomment-473973493. In particular, if Fetch isn't in control of setting these headers, you might be in for a surprise with service workers.

Reply:
I don't quite follow where that conversation ended up. What's the behavior in Service Workers that we want? What behavior would surprise developers? @yoavweiss

Reply:
So, the surprising bits would be that `Sec-` prefixed headers would disappear from the requests when developers would modify the request in SW in any way (but won't disappear for pass-through cases).

For the CH case, I think we could solve this by re-adding them below SWs (but @annevk had reservations). For Fetch-Metadata, maybe you could simply add them below SWs?

Reply:
I think for Fetch-Metadata all is in order because they are set as part of main fetch.